pkg/session/serve.go

```diff
@@ -264,19 +269,45 @@ func (s *Session) serve() {
 
 		case "triggerComponentCheck":
 			if payload.ComponentName != "" {
-				comp := s.componentsRegistry.Get(payload.ComponentName)
-				if comp == nil {
-					log.Logger.Warnw("component not found", "name", payload.ComponentName)
-					response.ErrorCode = http.StatusNotFound
-					break
-				}
+				switch payload.TagName {
```

Comment:
we have to take some time to refactor the file... it's too long to maintenance..